.travis.yml

```diff
@@ -24,6 +24,8 @@ matrix:
       env: PYSPARK_PYTHON=python3 SPARK_VERSION=2.1.2 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
     - jdk: oraclejdk8
       env: PYSPARK_PYTHON=python3 SPARK_VERSION=2.2.1 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
+    - jdk: oraclejdk8
+      env: PYSPARK_PYTHON=python3 SPARK_VERSION=2.3.0 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
```

Comment:
can you add the python2 mix

Reply:
done

Reply:
can I make version 2.3 default in `build.sbt`:
`val sparkVer = sys.props.getOrElse("spark.version", "2.3.0")`?